index.html

```diff
@@ -36,29 +36,29 @@ <h1>Platform Engineering Maturity Model Assessment</h1>
                 <label class="option">
                   <input type="radio" name="investment_1" value="2" />
                   <div class="option-text">
-                    <span class="option-level">Technical team</span><br />
+                    <span class="option-level">Technical team(s)</span><br />
                     <span class="option-description"
                       >Dedicated platform team with general technical
-                      skills</span
+                      skills, or coordinated group of technical teams working together</span
```

Comment:
Is there any way to make this more concise... it's getting kinda long with this. It will end up having to be "as long as it needs to be", but I'm keen that it's "no longer", if that makes sense?

Reply:
Maybe: "Dedicated platform team or group of teams, with general technical skills"